AppHub/AppHub/AHBuild.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithBundle:(NSBundle *)bundle info:(NSDictionary *)info
     if ((self = [super init])) {
         _buildDescription = [info[AHBuildDataDescriptionKey] copy] ?: @"This build was downloaded from the App Store.";
         _bundle = bundle ?: [NSBundle mainBundle];
-        _compatibleIOSVersions = info[AHBuildDataCompatibleIOSVersionsKey] ?: AHShortVersionString();
+        _compatibleIOSVersions = [info[AHBuildDataCompatibleIOSVersionsKey] allValues] ?: @[AHShortVersionString()];
```

Comment:
Nice find here! The `AHShortVersionString()` is certainly a bug (this should be a `NSDictionary`). Could we change it to be like so?
```
 _compatibleIOSVersions = info[AHBuildDataCompatibleIOSVersionsKey] ?: @{AHShortVersionString(): AHShortVersionString()};
```

I'm afraid that `build.compatibleIOSVersions` is used elsewhere in the codebase as an `NSDictionary`, and making it return an `NSArray` would require some other changes, and also might break user code. Since this is a backward-incompatible change, we should probably change the behavior in a future patch.